android/source/VideoLocker/src/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -107,30 +108,37 @@ public void onClick(View v) {
             }
         });
 
-        if (Config.getInstance().getEnrollment().getEnabled()) {
+
             TextView tvFindCourses = (TextView) layout.findViewById(R.id.drawer_option_find_courses);
             tvFindCourses.setOnClickListener(new OnClickListener() {
                 @Override
                 public void onClick(View v) {
                     Activity act = getActivity();
                     ((BaseFragmentActivity) act).closeDrawer();
-
-                    if (!(act instanceof FindCoursesActivity)) {
-                        Router.getInstance().showFindCourses(act);
-
-                        //Finish need not be called if the current activity is MyCourseListing
-                        // as on returning back from FindCourses,
-                        // the student should be returned to the MyCourses screen
-                        if (!(act instanceof MyCoursesListActivity)) {
-                            act.finish();
+                    if (Config.getInstance().getEnrollment().getEnabled()) {
+                        if (!(act instanceof FindCoursesActivity)) {
+                            Router.getInstance().showFindCourses(act);
```

Comment:
@shahidtamboli When enrollment is disabled, 
that is when <code>Config.getInstance().getEnrollment().getEnabled()</code> returns false,
Shouldn't we show the UX popup window instead of find courses screen?  Please verify this.